phpstan.neon

```diff
@@ -22,7 +22,6 @@ parameters:
         - '%rootDir%/../../../src/Propel/Generator/Command/InitCommand.php'
     ignoreErrors:
         - '#Call to an undefined method .+Collection::.+Array\(\)#'
-        - '#Class .+TreeBuilder constructor invoked with 0 parameters, 1-3 required#'
```

Comment:
Would you mind explaining why is this necessary?

Reply:
because phpstan is complaining that this ignore-rule is not neccessary/used anymore: https://github.com/propelorm/Propel2/runs/3067690860

Reply:
Right, you removed the rule. I need to check my color perception. Sorry!